src/app/components/StakingDateSelector/CustomButtonGroup.tsx

```diff
@@ -0,0 +1,52 @@
+import classNames from 'classnames';
+import React, { useMemo } from 'react';
+import { ButtonGroupProps } from 'react-multi-carousel';
+import { ArrowType, CustomArrow } from './CustomArrow';
+import styles from './index.module.scss';
+
+export const CustomButtonGroup = ({
+  next,
+  previous,
+  ...rest
+}: ButtonGroupProps) => {
+  const { carouselState } = rest;
+  const totalItems = carouselState?.totalItems || 0;
+
+  const isFirstSlide = useMemo(() => carouselState?.currentSlide === 0, [
+    carouselState?.currentSlide,
+  ]);
+
+  const isLastSlide = useMemo(
+    () => carouselState?.slidesToShow === carouselState?.currentSlide,
+    [carouselState?.currentSlide, carouselState?.slidesToShow],
+  );
+
+  const isNotPaginated = useMemo(
+    () =>
+      carouselState && carouselState.slidesToShow > carouselState.totalItems,
```

Comment:
This reads like an apples and oranges comparison, are you sure this is correct?

Reply:
It may seem weird but this is probably the best way to find if there should be a carousel at all. 

Basically, it means that if the total number of items is less (I adjusted it to less or equal in the latest commit) than the number of items I want to display in one step of a slider, don't display the arrows.